archunit/src/main/java/com/tngtech/archunit/core/domain/JavaClass.java

```diff
@@ -659,6 +659,11 @@ public boolean isMetaAnnotatedWith(DescribedPredicate<? super JavaAnnotation<?>>
         return members.getReferencedClassObjects();
     }
 
+    @PublicAPI(usage = ACCESS)
+    public Set<TypeCast> getypeCast() {
```

Comment:
Maybe `getTypeCast`**`s`**? (And same in `JavaMembers`.)

Reply:
This method is currently called `getypeCast`. As it returns a `Set` of `TypeCast` objects, I thought that a pluralized name `getTypeCasts` might be more appropriate.

The delegation to `JavaMembers` is probably fine, but I'd also rename the method there.